server/models/Collective.js

```diff
@@ -225,6 +226,16 @@ export default function(Sequelize, DataTypes) {
 
       address: DataTypes.STRING,
 
+      countryISO: {
+        type: DataTypes.STRING,
+        validate: {
+          len: 2,
+          isCountryISO(value) {
+            return isISO31661Alpha2(value);
```

Comment:
I'm surprised by this: documentation state that the validate function is supposed to throw, not to return a boolean. Did you check that this was working properly?

Reply:
If we have a doubt about it, would be great to back that with tests.

Reply:
Apologies! :slightly_smiling_face: